src/lib/rocksdb/database.ml

```diff
@@ -165,10 +165,6 @@ let%test_unit "checkpoint read" =
                 ~expect:cp_sorted cp_alist ;
               close db ;
               close cp ;
-              let%bind () = File_system.remove_dir db_dir in
```

Comment:
Do we need to remove these calls as part of this upgrade?

Reply:
I'd removed them because we get: `pthread lock: Invalid argument`.

I'll see if I can work around that.

Reply:
After removing these calls, and running the unit test, the temp directories no longer exist in `/tmp`. Could it be the unit test framework knows to do this?